python/sglang/srt/server_args.py

```
@@ -709,6 +710,12 @@ def add_cli_args(parser: argparse.ArgumentParser):
             default=ServerArgs.tokenizer_path,
             help="The path of the tokenizer.",
         )
+        parser.add_argument(
+            "--tokenizer-worker-num",
```

i couldnt find the validation, nit
validate it and fail fast, not a must have tho

---

resolved